### ISSUE TITLE
Description of a sparring kumite is difficult to keep up to date

### ISSUE THREAD

**Issue by hobovsky** [Codewars/codewars.com#3156]
When playing around with kumite ([here](https://www.codewars.com/kumite/65b8301c8d67ab2ae49b5829?sel=65b8301c8d67ab2ae49b5829)) with other users, we noticed that it is difficult to keep its description up to date as the kumite progresses:
- The description is not forked. When you fork a kumite, the descritpion field in the new fork is empty.
- After a fork is published, the parent fork cannot be edited anymore, and the description of the first fork of the kumite cannot be updated.
- There is no easy way to view and/or copy markdown of a description from an ancestor fork to paste it into a new fork.

Basically the only feasible option is to keep a description somewhere offline, edit it locally when needed, and paste in new forks, which is not a great workflow.
